### PR TITLE
{bio}[intel/2019b] Flye v2.6 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/Flye/Flye-2.6-intel-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://github.com/fenderglass/Flye'
+description = """Flye is a de novo assembler for long and noisy reads, such as those produced by PacBio
+ and Oxford Nanopore Technologies."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+source_urls = ['http://github.com/fenderglass/Flye/archive/']
+sources = ['%(version)s.tar.gz']
```
Comment (verdurin) [inline easybuild/easyconfigs/f/Flye/Flye-2.6-intel-2019b-Python-3.7.4.eb]: @boegel Please add the missing checksum.

Comment (boegel) [reply easybuild/easyconfigs/f/Flye/Flye-2.6-intel-2019b-Python-3.7.4.eb]: fixed in 0018fc7

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/8c1dde825bac82ae8efc36fec6803c36 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/bb9247d4cb605c50c954ff8026cdb73e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ba9211634a2257a391855c0138173ad7 for a full test report.

**Comment by bartoldeman** [comment]
Going in, thanks @boegel!

**Pull request merged**